### DISCUSSION TITLE
firefox: crash reports are missing

### DISCUSSION THREAD

**Discussion by WhyNotHugo** [netblue30/firejail#5264] [Q&A] [answered]
`about:crashes` shows nothing from the last year. I think all the old crashes are from some setup  I had before firejail.

Any ideas how to get these to save?

**Comment by rusty-snake** [comment]
Do you trigger firefox chrash handling if you SIGKILL it? Or do you know an other way to test this?

Try add `allow-debuggers` and see if it works.

**Reply by WhyNotHugo** [reply]
Firefox crashes any time I try to drag ANY tab.

I was asked for a crash report to figure out what's wrong here.

https://www.reddit.com/r/firefox/comments/w2qqey/comment/igsfy72/?context=3

**Comment by rusty-snake** [comment, marked as answer]
Also search for duplicates before asking.

#2212

**Reply by WhyNotHugo** [reply]
I searched for `about:crashes`, `firefox crashes`, `firefox crash reports`. I looked at `firefox.local`. Nothing relevant. Too many unrelated reports.

**Reply by WhyNotHugo** [reply]
Can't I use `coredumpctl` to generate a "crash report"?

**Reply by rusty-snake** [reply]
![Screenshot-fs8](https://user-images.githubusercontent.com/41237666/179836267-afb78a7a-b565-4a0f-a9bd-5cba20e28403.png)

**Reply by rusty-snake** [reply]
> Can't I use coredumpctl to generate a "crash report"?

Depening on what a "crash report" is/should contain, yes.
